docs/rules/0191/ruby-package.md

```diff
@@ -0,0 +1,101 @@
+---
+rule:
+  aip: 191
+  name: [core, '0191', ruby-package]
+  summary: The `option ruby_package` annotation should be idiomatic if set.
+permalink: /191/ruby-package
+redirect_from:
+  - /0191/ruby-package
+---
+
+# Ruby package annotation
+
+This rule enforces that if a proto file for a public API surface sets
+`option ruby_package`, that it uses language idiomatic conventions, as mandated
+in [AIP-191][].
+
+## Details
+
+This rule looks at each proto file, and complains if the `ruby_package` file
+annotation uses anything other than upper camel case, or includes characters
+other than letters, numbers, and `:`.
+
+It also ensures that versions with stability (e.g. `V1Beta1`) are capitalized
+appropriately.
+
+## Examples
+
+### Case
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+syntax = "proto3";
+
+package google.example.v1;
+
+option ruby_package = "google.example.v1";
+```
+
+```proto
+// Incorrect.
+syntax = "proto3";
+
+package google.example.v1;
+
+option ruby_package = "Google.Example.V1";
+```
+
+**Correct** code for this rule:
+
+```proto
+// Correct.
+syntax = "proto3";
+
+package google.example.v1;
+
+option ruby_package = "Google::Example::V1";
+```
+
+### Versions with stability
+
+**Incorrect** code for this rule:
+
+```proto
+// Incorrect.
+syntax = "proto3";
+
+package google.example.v1beta1;
+
+option ruby_package = "Google::Example::V1beta1"; // Should be V1Beta1.
```

Comment:
Actually, it _should_ be `V1beta1`, and _not_ `V1Beta`.

Reply:
Thanks!

Reply:
Fixed.